pkg/rancher-desktop/backend/lima.ts

```diff
@@ -1780,9 +1780,7 @@ export default class LimaBackend extends events.EventEmitter implements VMBacken
       try {
         const env: NodeJS.ProcessEnv = {};
 
-        if (this.cfg?.experimental.virtualMachine.sshPortForwarder) {
-          env.LIMA_SSH_PORT_FORWARDER = 'true';
-        }
+        env.LIMA_SSH_PORT_FORWARDER = this.cfg?.experimental.virtualMachine.sshPortForwarder ? 'false' : 'true';
```

Comment:
Isn't this the wrong way?

```suggestion
        env.LIMA_SSH_PORT_FORWARDER = this.cfg?.experimental.virtualMachine.sshPortForwarder ? 'true' : 'false';
```

Reply:
You're right, my brain was stuck on "use ssh forwarder means disable GRPC". Will fix.